sfdx-source/apex-common-samplecode/main/classes/domains/Opportunities.cls

```diff
@@ -39,7 +39,7 @@ public class Opportunities extends fflib_SObjectDomain
 	
 	public Opportunities(List<Opportunity> sObjectList)
 	{
-		// Domain classes are initialised with lists to enforce bulkification throughout
+		// Domain classes are initialised with lists to enforce nullification throughout
```

Comment:
@daveespo is this change correct?  I would have expected "bulkification" to be the correct word here.

Reply:
I think you are right.. Let me fix that.